dask_labextension/manager.py

```diff
@@ -264,16 +264,21 @@ def make_cluster_model(
     """
     # This would be a great target for a dataclass
     # once python 3.7 is in wider use.
+    try:
+        info = cluster.scheduler_info
+    except AttributeError:
+        info = cluster.scheduler.identity()
+    assert isinstance(info, dict)
     model = dict(
         id=cluster_id,
         name=cluster_name,
         scheduler_address=cluster.scheduler_address,
         dashboard_link=cluster.dashboard_link or "",
-        workers=len(cluster.scheduler.workers),
+        workers=len(info["workers"]),
         memory=utils.format_bytes(
-            sum(ws.memory_limit for ws in cluster.scheduler.workers.values())
+            sum(d["memory_limit"] for d in info["workers"].values())
         ),
-        cores=sum(ws.ncores for ws in cluster.scheduler.workers.values()),
+        cores=sum(d["nthreads"] for d in info["workers"].values()),
```

Comment:
What is the intended compatibility profile for this? I am getting failures here due to `nthreads` not being available in my workers with dask v1.2 and distributed v1.28.

Reply:
2.0+, but it's fairly easy to go handle this to go back further.  I'll push up a small fix.

Reply:
:+1: I'd be in favor of maintaining 1.0 compatibility, if it's a small check.